pkg/cli/sql.go

```diff
@@ -1042,6 +1043,10 @@ func (c *cliState) doHandleCliCmd(loopState, nextState cliStateEnum) cliStateEnu
 		c.concatLines = `SHOW TABLES`
 		return cliRunStatement
 
+	case `\dT`:
+		c.concatLines = `SHOW ENUMS`
```

Comment:
This makes me a little sad: `\dT` should be about user-defined types; why restrict it to enums?

There should be a `SHOW TYPES` in SQL that does "the right thing", and used here. Maybe it's time to add it preemptively?

Reply:
Yeah i think that makes sense. I'll add it.